src/db.rs

```diff
@@ -40,16 +40,20 @@ pub struct DynField {
 )]
 pub enum DynFieldKind {
 	#[default]
+	TextLineSecret,
 	TextLine,
-	SecretLine,
+	MultiLine,
+	MultiLineSecret,
```

Comment:
Renaming those as they didn't make any sense not with a multiline field existing